### PR TITLE
Add deep lab cut interface 

### PR DESCRIPTION
**Pull request by h-mayorquin**
This a re-hash of #414 and should close #414.

I changed the `run_conversion` signature to the new style.

### CODE REVIEW

File: docs/conversion_examples_gallery/behavior/deeplabcut.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    >>> file_path = BEHAVIOR_DATA_PATH / "DLC" / "m3v1mp4DLC_resnet50_openfieldAug20shuffle1_30000.h5"
+    >>> config_file_path = BEHAVIOR_DATA_PATH / "DLC" / "config.yaml"
+    >>> subject_name = "ind1"
+    >>>
+    >>> interface = DeepLabCutInterface(file_path=file_path, config_file_path=config_file_path, subject_name=subject_name, verbose=False)
+    >>> metadata = interface.get_metadata()
+    >>> metadata.update(NWBFile=dict())
+    >>> # For data provenance we add the time zone information to the conversion
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata["NWBFile"].update(session_start_time=session_start_time)
```
Comment (bendichter) [inline docs/conversion_examples_gallery/behavior/deeplabcut.rst:22]:
```suggestion
    >>> # NWB requires a session start time
    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
    >>> metadata.get("NWBFile", dict()).update(session_start_time=session_start_time)
```

File: docs/conversion_examples_gallery/behavior/deeplabcut.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    >>> # Choose a path for saving the nwb file and run the conversion
+    >>> nwbfile_path = f"{path_to_save_nwbfile}"
+    >>> interface.run_conversion(nwbfile_path=nwbfile_path, metadata=metadata)
```
Comment (bendichter) [inline docs/conversion_examples_gallery/behavior/deeplabcut.rst:25]:
```suggestion
    >>> interface.run_conversion(nwbfile_path="path_to_save_nwbfile.nwb", metadata=metadata)
```

**Comment by h-mayorquin** [comment]
I will move this to the new repo.

File: docs/conversion_examples_gallery/behavior/deeplabcut.rst (hunk 2 above)
Comment (h-mayorquin) [reply docs/conversion_examples_gallery/behavior/deeplabcut.rst:25]: This parameter needs to be used later as a test that the file was created.

**Comment by h-mayorquin** [comment]
I will close this as the PR in the new repo is the one were the latest updates have been pushed and is now mostly ready.

**Pull request closed without merging**